Example/Sources/ConversationViewController.swift

```diff
@@ -124,6 +139,13 @@ class ConversationViewController: MessagesViewController {
         ]
         items.forEach { $0.tintColor = .lightGray }
         
+        messageInputBar.attachmentManager.addAttachmentCellPressedBlock = { [weak self] _ in
+            let imagePicker = UIImagePickerController()
+            imagePicker.delegate = self
+            imagePicker.sourceType = .photoLibrary
+            self?.present(imagePicker, animated: true, completion: nil)
```

Comment:
I recall instances where the ImagePickerController gets deallocated after it's dismissed because there's no strong reference to it outside of the block. Have you seen that? Speaking from hazy memory when adding a UIAlertAction that presents UIImagePickerController.